### DISCUSSION TITLE
Using the clipboard with Ctrl+Insert and Shift+Insert doesn't work in osu!'s text fields

### DISCUSSION THREAD

**Discussion by soopax** [ppy/osu#24017] [Q&A] [answered]
As an example this is listed here https://support.microsoft.com/en-us/windows/keyboard-shortcuts-in-windows-dcc61a57-8ff0-cffe-9796-cb9706c75eec as a "general keyboard shortcut". Not very important but it would be nice to be able to use this in osu!.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu-framework/pull/5870
